### PR TITLE
[SALEOR-6645] Show appUrl in 3rd-party apps list

### PR DESCRIPTION
**Pull request by magul**
I want to merge this change because we want to show `appUrl` on a list of 3rd-party apps to differentiate app installed during development.

### Screenshots
Screenshot 2022-04-26 at 15 01 59

### Pull Request Checklist

1. [X] This code contains UI changes
2. [X] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6645 Add app URL](https://app.clickup.com/t/2549495/SALEOR-6645)

**Comment by magul** [comment]
# WARNING
It will require https://github.com/saleor/macaw-ui/pull/137 to be merged, released and upgraded in dashboard.

File: src/apps/components/AppUrlLink/AppUrlLink.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { LinkIcon, Pill } from "@saleor/macaw-ui";
+import React from "react";
+
+interface AppUrlLinkProps {
+  url: string | null;
+  onClick: () => void;
+}
+
+const AppUrlLink: React.FC<AppUrlLinkProps> = ({ url, onClick }) => {
+  if (typeof url === "string") {
```
Comment (orzechdev) [inline src/apps/components/AppUrlLink/AppUrlLink.tsx:10]: Why not just the following?
```suggestion
  if (url) {
```

File: src/apps/components/AppUrlLink/AppUrlLink.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        label={url as string}
+        color="info"
+        size="small"
+        onClick={onClick}
+      />
+    );
+  }
+  return (
+    <Pill
+      label="No app url defined"
```
Comment (orzechdev) [inline src/apps/components/AppUrlLink/AppUrlLink.tsx:23]: Please translate it with intl

File: src/apps/components/AppUrlLink/AppUrlLink.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { LinkIcon, Pill } from "@saleor/macaw-ui";
+import React from "react";
+
+interface AppUrlLinkProps {
+  url: string | null;
+  onClick: () => void;
+}
+
+const AppUrlLink: React.FC<AppUrlLinkProps> = ({ url, onClick }) => {
```
Comment (orzechdev) [inline src/apps/components/AppUrlLink/AppUrlLink.tsx:9]: I'm not sure if the name is the best possible. What about `AppPillLink` or making component more generic with name like `PillLink`?

**Review by orzechdev — CHANGES_REQUESTED** [review]

File: src/apps/components/AppUrlLink/AppUrlLink.tsx (hunk 3 above)
Comment (dominik-zeglen) [reply src/apps/components/AppUrlLink/AppUrlLink.tsx:9]: I think it's okay, it handles urls and has goto icon, so only improvement i see would be `UrlPill`, `PillWithUrl` or something like this. But `AppUrlLink` is okay too i guess

File: src/apps/components/AppUrlLink/AppUrlLink.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  if (typeof url === "string") {
+    return (
+      <Pill
+        icon={<LinkIcon />}
+        label={url as string}
+        color="info"
+        size="small"
+        onClick={onClick}
+      />
```
Comment (dominik-zeglen) [inline src/apps/components/AppUrlLink/AppUrlLink.tsx:18]: I you want to right-click action it you can try wrapping it with `Link` from `react-router-dom` or setting `component={Link} to={url}`

**Review by dominik-zeglen — APPROVED** [review]

**Review by Cloud11PL — APPROVED** [review]

**Pull request merged**